pkg/gds/admin_test.go

```diff
@@ -643,6 +648,7 @@ func (s *gdsTestSuite) TestRetrieveVASP() {
 func (s *gdsTestSuite) TestUpdateVASP() {
 	s.LoadSmallFixtures()
 	defer s.ResetSmallFixtures()
+	defer emails.PurgeMockEmails()
```

Comment:
Got left behind from refactor - I don't think UpdateVASP sends emails anymore - but not a big deal.